### PR TITLE
DM-17071: Remove afw.math.mathLib

### PR DESCRIPTION
**Pull request by kfindeisen**

### CODE REVIEW

**Review by kfindeisen — COMMENTED** [review]
I think the addition of underscores to some of the class/function names will confuse anyone looking at this code in the future. Once that's fixed, I think it's good to merge.

File: python/lsst/afw/math/warper.py [hunk 1]
```diff
@@ -24,7 +24,9 @@
 import lsst.pex.config as pexConfig
 import lsst.geom
 import lsst.afw.image as afwImage
-from . import mathLib
+from .warpExposure import warpExposure as _warpExposure
+from .warpExposure import warpImage as _warpImage
+from .warpExposure import WarpingControl as _warpingControl
```
[the remark this answers is not in the document]
Comment (mwittgen) [reply python/lsst/afw/math/warper.py:29]: warpImage is also declared as function in warper.py

[the remark this answers is not in the document]
Comment (kfindeisen) [reply python/lsst/afw/math/warper.py:29]: Then I think the best solution is to not use `from ... import` at all, and have the code read e.g. `.warpExposure.warpImage`.

[the remark this answers is not in the document]
Comment (mwittgen) [reply python/lsst/afw/math/warper.py]: Was was wrong about warpImage. Pushed the updates with squashed commits.

**Pull request merged**